### PR TITLE
[CAM8421] add the filters explanation to the Activity section

### PR DESCRIPTION
**Pull request by aryaroudi**
I found that the section for explaining the filters is missing in KB so I added that part to activity section

### CODE REVIEW

File: Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md [hunk 1]
```diff
@@ -116,10 +116,17 @@ ___
 
 ![getting-started-login-10.png](../../images/cloud-application-manager/getting-started-login-10.png)
 
-* Allows you to see (based on your scope and permissions) all of the relevant activity that has occured in the platform, the type of event and the user that performed the event.
+* Allows you to see (based on your scope and permissions) all of the relevant activity that has occurred in the platform, the type of event and the user that performed the event.
 * Filter through activity by type of event or switch between all activity or current scope.
+![activity-filter-list.png](../../images/cloud-application-manager/activity-filter-list.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md]:
```suggestion
![Activity page, Filter dropdown](../../images/cloud-application-manager/activity-filter-list.png)
```

**Comment by gsanchezu** [comment]
Please, update TOC with links and Help section. Add common sections (Overview, Audience, Prereqs) and change ALL image alternate texts to something more meaningful.

**Comment by jim-phillips** [comment]
@aryaroudi waiting on changes per @gsanchezu

**Comment by aryaroudi** [comment]
> @aryaroudi waiting on changes per @gsanchezu

Hi @jim-phillips the the requirment by @gsanchezu has been implemented.

Thanks

File: Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md [hunk 2]
```diff
@@ -116,10 +129,17 @@ ___
 
 ![getting-started-login-10.png](../../images/cloud-application-manager/getting-started-login-10.png)
```
Comment (gsanchezu) [inline Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md]:
```suggestion
![Activity page](../../images/cloud-application-manager/getting-started-login-10.png)
```

**Pull request merged**